wasm_runtime/src/c_api.rs

```diff
@@ -468,3 +468,162 @@ pub extern "C" fn wasm_executionctx_run(executionctx_id: *const c_char, _buffer:
 pub extern "C" fn wasm_return_const_char_ownership(ptr: *const c_char) {
     deallocate_cstring(ptr);
 }
+
+
+// ##########################################################################
+//                              Unit Tests
+// ##########################################################################
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::ffi::CString;
+    use crate::config::WASM_RUNTIME_CONFIGS;
+
+    #[test]
+    fn wasm_wasm_module_load() {
+        // setup
+        let wasm_path = CString::new("../examples/wasm_modules/rust-wasm/hello_wasm.wasm").unwrap();
+        let nonexistent_path = CString::new("../attempt/to/load/me.wasm").unwrap();
+
+        // test success
+        let mut val = wasm_module_load(wasm_path.as_ptr());
+        assert_eq!(val, 0);
+
+        // test failure
+        val = wasm_module_load(nonexistent_path.as_ptr());
+        assert_eq!(val, -1);
+    }
+
+    #[test]
+    fn wasm_wasm_config_create() {
```

Comment:
```suggestion
    fn wasm_config_create_general() {
```

Reply:
Based on your exchange with `ereslibre` I have changed the `_general` suffix to `_success` where applicable